docs/.babelrc.json

```diff
@@ -1,3 +1,11 @@
 {
-  "presets": ["@docusaurus/core/lib/babel/preset"]
+  "presets": ["@docusaurus/core/lib/babel/preset"],
+  "browserslist": {
+    "production": [">0.5%", "not dead", "not op_mini all"],
```

Comment:
It means we support IE11.
```
$  npx browserslist ">0.5%, not dead, not op_mini all"
and_chr 94
and_uc 12.12
chrome 94
chrome 93
chrome 92
chrome 91
edge 94
edge 93
edge 92
firefox 92
firefox 91
ie 11
ios_saf 15
ios_saf 14.5-14.8
ios_saf 14.0-14.4
ios_saf 12.2-12.5
opera 78
safari 14.1
safari 13.1
samsung 15.0
```

Can we support just 2 last versions of every major browser both in prod and dev.
```
last 2 Chrome versions
last 2 iOS versions
last 2 Safari versions
last 2 Firefox versions
last 2 Opera versions
last 2 Edge versions
```
I think it would be more predictable and simpler.

Reply:
@saihaj `last 2 versions` still include IE
```
$ npx browserslist "last 2 versions"
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
and_chr 87
and_ff 83
and_qq 10.4
and_uc 12.12
android 81
baidu 7.12
bb 10
bb 7
chrome 87
chrome 86
edge 87
edge 86
firefox 84
firefox 83
ie 11
ie 10
ie_mob 11
ie_mob 10
ios_saf 14.0-14.3
ios_saf 13.4-13.7
kaios 2.5
op_mini all
op_mob 59
op_mob 12.1
opera 72
opera 71
safari 14
safari 13.1
samsung 13.0
samsung 12.0
```

Please use the config I suggested above ⬆️, also please use it for both prod and dev.

Reply:
Maintaining this list is more painful.

Reply:
If there is an issue that comes up with IE we can take care then.

Reply:
How about
```
$ npx browserslist ">0.5%, not dead, not op_mini all, not ie > 0"
and_chr 94
and_uc 12.12
chrome 94
chrome 93
chrome 92
chrome 91
edge 94
edge 93
edge 92
firefox 92
firefox 91
ios_saf 15
ios_saf 14.5-14.8
ios_saf 14.0-14.4
ios_saf 12.2-12.5
opera 78
safari 14.1
safari 13.1
samsung 15.0
```
It's the same as docusaurus defaults but without IE?

Reply:
I don't want to hardcode these. Having just range is fine cause it will automatically keep it update to date.